packages/google-pay-integration/src/google-pay-payment-strategy.ts

```diff
@@ -185,12 +188,46 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
         );
     }
 
+    protected async _isSignedIn(): Promise<boolean> {
+        let signedIn = false;
+
+        try {
+            signedIn = !!(await this._googlePayPaymentProcessor.getNonce(this._getMethodId()));
```

Comment:
We have different implementation of nonce for a few strategies. Can you please take a look at the few following strategies (google-pay-adyenv3-gateway, google-pay-adyenv3-gateway, google-pay-braintree-gateway) to be sure that this changes is applicable for each of them pls?